scripts/storage-emulator-integration/tests.ts

```diff
@@ -1309,6 +1309,29 @@ describe("Storage emulator", () => {
             expect(md.downloadTokens.split(",")).to.not.deep.equal([token]);
           });
       });
+
+      it("#uploadResumableDoesNotRequireMultipleAuthHeaders", async () => {
+        const uploadURL = await supertest(STORAGE_EMULATOR_HOST)
+          .post(
+            `/v0/b/${storageBucket}/o/test_upload.jpg?uploadType=resumable&name=test_upload.jpg`
+          )
+          .set({
+            Authorization: "Bearer owner",
+            "X-Goog-Upload-Protocol": "resumable",
+            "X-Goog-Upload-Command": "start",
+          })
+          .expect(200)
+          .then((res) => new URL(res.header["x-goog-upload-url"]));
+
+        await supertest(STORAGE_EMULATOR_HOST)
```

Comment:
could you include an additional expect with `upload` and no `finalize`? Multipart uploads can have unbounded numbers of upload requests but only one start and one finalize

Reply:
Done.